react-hot-button-component/my-app/src/HotButton.js

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react';
+
+export default function HotButton () {
+  const allColors = ['#000000', '#351C74', '#674DA7', '#E06666', '#F6B26A', '#FEFF00', '#FFFFFF'];
+  let [numClicks, setNumClicks] = useState(0);
+  const bgColor = (numClicks % 3 === 0) ? allColors[numClicks/3] : allColors[Math.floor(numClicks / 3)];
```

Comment:
Clever! I didn't notice the pattern in the click counts!

I think you can just use `const bgColor = allColors[Math.floor(numClicks / 3)];` can't you?